requests/__init__.py

```diff
@@ -86,6 +87,25 @@ def check_compatibility(urllib3_version, chardet_version):
 except ImportError:
     pass
 
+# Check for older version of pyOpenSSL
+def check_pyopenssl(cryptography_version, pyopenssl_version):
```

Comment:
This should be a private function. We really shouldn't be exposing this as part of our API.

Reply:
`check_compatibility()` defined just above `check_pyopenssl` wasn't private, so I thought it would be best to expose this too for the sake of consistency.